babel.config.js

```diff
@@ -29,7 +29,7 @@ const minimist = require('minimist');
 const {isTravisBuild} = require('./build-system/common/travis');
 const argv = minimist(process.argv.slice(2));
 
-const isDist = argv._.includes('dist');
+const isDist = argv._.includes('dist') || argv._.includes('check-types');
```

Comment:
Since this now includes `dist` and `check-types`, maybe rename `isDist` to `usesClosureCompiler`?

Reply:
done.